### DISCUSSION TITLE
Multiple subscription plans

### DISCUSSION THREAD

**Discussion by frederikme** [Glench/ExtPay#85] [Ideas]
Hi, thanks for this wonderful work you're doing!

Is there a possibility to offer multiple subscription plans (for example 5$, 10$, 20$) and then retrieve which subscription plan the user has paid for in the extension to provide the services accordingly?

Many thanks

**Comment by akump** [comment]
He's already working on this

**Reply by frederikme** [reply]
Ok thanks!

**Comment by davidgarry1** [comment]
I also would love this feature!
